src/result_buffer.jl

```diff
@@ -15,11 +34,30 @@ module RowStatus
     const UnknownTypeError    = 0x10 # We couldn't parse some fields because we don't know how to parse any instance of that type
     const SkippedRow          = 0x20 # The row contains no valid values
 
+    # Used in DebugContext
     const Marks = ('✓', '?', '<', '>', '!', 'T', '#')
     const Names = ("Ok", "HasColumnIndicators", "TooFewColumns", "TooManyColumns", "ValueParsingError", "UnknownTypeError", "SkippedRow")
     const Flags = (0x00, 0x01, 0x02, 0x04, 0x08, 0x10, 0x20)
 end
 
+"""
+    BitSetMatrix <: AbstractMatrix{Bool}
+
+A matrix representing the missingness of values in the result buffer.
+The matrix has as many rows, as there are rows with at least one missing value in the result buffer,
+but the number of columns is always the same.
+
+It is this recommended to iterate the result buffer from start to finish and note the `RowStatus` for
```

Comment:
when would one do this? when implementing a custom result buffer? or when consuming a result buffer? or?

Reply:
Good point, I'll add "When consuming ..." to clarify the context.